operations/provision-consul/k8s/README.md

```diff
@@ -0,0 +1,476 @@
+
+    kubectl cluster-info
+
+    Kubernetes master is running at https://35.197.4.209
+    GLBCDefaultBackend is running at https://35.197.4.209/api/v1/namespaces/kube-system/services/default-http-backend:http/proxy
+    Heapster is running at https://35.197.4.209/api/v1/namespaces/kube-system/services/heapster/proxy
+    KubeDNS is running at https://35.197.4.209/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+    Metrics-server is running at https://35.197.4.209/api/v1/namespaces/kube-system/services/https:metrics-server:/proxy
+    
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+
+    kubectl version
+
+    Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.2", GitCommit:"bb9ffb1654d4a729bb4cec18ff088eacc153c239", GitTreeState:"clean", BuildDate:"2018-08-07T23:17:28Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
+    Server Version: version.Info{Major:"1", Minor:"10+", GitVersion:"v1.10.6-gke.2", GitCommit:"384b4eaa132ca9a295fcb3e5dfc74062b257e7df", GitTreeState:"clean", BuildDate:"2018-08-15T00:10:14Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}
+
+
+### Validating Helm's Tiller on the Kubernetes cluster
+
+This guide also assumes that Helm's [Tiller](https://docs.helm.sh/glossary/#tiller) is installed and functioning properly on the k8s cluster. The [Helm website](https://helm.sh)
+has documentation on how to install and configure Helm and Helm Tiller.
+
+To validate a working Tiller (assumes the default of Tiller running in the Kubernets 'kube-system' namespace):
+
+    kubectl get pods --namespace=kube-system | grep tiller
+
+    tiller-deploy-895d57dd9-q82bp                                   1/1       Running   0          14d
+
+    helm version
+
+    Client: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
+    Server: &version.Version{SemVer:"v2.10.0", GitCommit:"9ad53aac42165a5fadc6c87be0dea6b115f93090", GitTreeState:"clean"}
+
+Your versions my differ slightly but so long as you have Running Tiller and the Client and Server versions are
+reporing reasonable values you are probably in good shape.
+
+
+### Download and install the official HashiCorp Consul Helm Chart
+
+There are a number of ways to install a Helm Chart but the process changes slightly from version to version of Helm
+and the workflow is different during Helm Chart development versus production usage. To keep things simple,
+we'll simply clone the Consul chart repo locally and drive the process from there. This is more similar to
+a development workflow but it lets us understand the workings of the Consul Helm Chart without diving too deep into
+Helm Chat workflow.
+
+    # FIXME: change this once the repo is public available
+    # git clone https://github.com/hashicorp/consul-helm ./consul-helm
+    git clone --progress git@github.com:hashicorp/consul-helm ./consul
```

Comment:
I would then recommend checking out a tagged release, master is not recommended.